packages/empty-state/src/EmptyState.js

```diff
@@ -83,9 +83,10 @@ class EmptyState extends React.PureComponent {
 
     return (
       <div className="buttons">
-        {actions.map((buttonProps, index) => (
+        {actions.map(buttonProps => (
           <Button
-            key={`empty-state-button-${index}`}
+            key={`empty-state-button-${buttonProps.text}`}
+            id={`empty-state-button-${buttonProps.text}`}
```

Comment:
Is this id for QA automated tests?
We could use `data-testid` so it would also work with React Testing Library.

Reply:
yes, ok i'll change it